### ISSUE TITLE
Version 0.0.7 reports as 0.0.6 in flake8

### ISSUE THREAD

**Issue by SebDieBln** [nhoad/flake8-unused-arguments#8]
To see a list of the installed ```flake8``` plug-ins you can run
```
flake8 --version
```
It prints a list of the installed plug-ins along with their version numbers. **The new release 0.0.7 reports as 0.0.6 in this list.**
```
> flake8 --version
3.9.1 (flake8-unused-arguments: 0.0.6, ...) CPython 3.9.4 on Windows
```
but inspecting the installed packages with ```pip``` we see the new version is actually installed:
```
> pip show flake8-unused-arguments
Name: flake8-unused-arguments
Version: 0.0.7
.
.
.
```

**Comment by nhoad** [comment]
Ah! I've probably forgotten to bump a number somewhere. I can take a look in probably four to six hours and get a new version out with the fix.

**Comment by nhoad** [comment]
Fixed! I've put out v0.0.8 now:

```
$ flake8 --version
3.9.1 (flake8-unused-arguments: 0.0.8, ...) CPython 3.9.4 on Darwin
```

**Comment by SebDieBln** [comment]
> 
> 
> Fixed! I've put out v0.0.8 now:

Thanks for the quick response!
